MapboxNavigation/RouteMapViewController.swift

```diff
@@ -330,7 +330,11 @@ class RouteMapViewController: UIViewController {
     @objc func didReroute(notification: NSNotification) {
         guard self.isViewLoaded else { return }
         
-        if !(routeController.locationManager is SimulatedLocationManager) {
+        if let locationManager = routeController.locationManager as? SimulatedLocationManager {
```

Comment:
Since we set this string in a couple of places, I'd like to see it exposed as a constant `NSLocalizedString`.

Reply:
@JThramer `value: "Simulating Navigation at %f×"` will not localize. Unfortunately, it's imperative we have it this way.

Reply:
What about a `NSLocalizedString.simulatingNavigation` extension property?

Reply:
@JThramer Sure! FYI, you cannot create an extension on `NSLocalizedString`. Instead, we can create an extension on String that returns an `NSLocalizedString`.

Reply:
NSLocalizedString isn’t a type, it’s a macro (that gets bridged to Swift as a function).

Reply:
@1ec5 yeah, I had forgotten that. We ended up using this `String.Localized` pattern which I think is pretty darn clean anyway, so all turned out well 😅